src/System.Globalization/tests/System/Globalization/TextInfoTests.cs

```diff
@@ -29,7 +29,11 @@ public void MiscTest(string cultureName, int lcid, int ansiCodePage, int ebcdiCC
             Assert.Equal(lcid, ti.LCID);
             Assert.Equal(ansiCodePage, ti.ANSICodePage);
             Assert.Equal(ebcdiCCodePage, ti.EBCDICCodePage);
-            Assert.Equal(macCodePage, ti.MacCodePage);
+            // On Nano Server >= 1809 the MacCodePage always returns 65001.
+            if (!PlatformDetection.IsWindowsNanoServer)
+            {
+                Assert.Equal(macCodePage, ti.MacCodePage);
```

Comment:
This is not the one you want to fix, I think is EBCDICCodePage, the one just above.

Reply:
No actually it's both of them

Reply:
It might even be all of them. That is why @tarekgh suggested me to just skip all the test.

Reply:
Also, we covering the test in other Windows specific test too. so it is not a big deal to disable this test for Nano.

Reply:
OK makes sense. Disabled it.